lib/scope.js

```diff
@@ -575,11 +584,10 @@ module.exports = {
     while ( prev_length > fields_to_set.length && fields_to_set.length > 0 ) {
 
       prev_length = fields_to_set.length;
-      // Go through each field on the page, seeing if you can set its
-      // variable
+      // Go through each field on the page, seeing if you can set its variable
       let remaining_fields = [];
-      for ( let i = 0; i < fields_to_set.length; i++ ) {
-        let field = fields_to_set[i];
+      for ( let field_i = 0; field_i < fields_to_set.length; field_i++ ) {
+        let field = fields_to_set[field_i];
```

Comment:
We could even go back to the old style loop here, since we don't need to use the index var `i` anymore (you're using the `var_data.indexOf(row)` function instead.

Reply:
Eh, this works too!